Inventario

Seba328 opened the pull request.

Seba328 committed:
Clase inventario. (Desarrollo)

Seba328 committed:
Merge branch 'developer' into Inventario

Seba328 committed:
Clase inventario.(Creado el fxml, inventarioView).

Seba328 committed:
Clase inventario.(Creado el fxml, inventarioView).++

Seba328 committed:
Clase inventario.(Creado el fxml, inventarioView).

Seba328 committed:
Cambio de nombre de Inventario a InventarioMaterial.

Seba328 committed:
Se agrega fecha de retiro

mrbarrientosg commented:
Esta todo perfecto

mrbarrientosg commented on src/cl/loccet/controller/InventarioMaterialController.java:
```diff
@@ -0,0 +1,47 @@
+        return materials;
+    }
+    public void agregarMaterial(Material material){
+        model.AgregarItem(material);
```
Aqui deberias verificar que el material cumpla todo antes de agregar al modelo, y eso seria todo

Seba328 committed:
heeeeelp

Seba328 committed:
Guardar proceso.

Seba328 committed:
Guardar proceso(2).

Seba328 committed:
guardar.

Seba328 committed:
Inventario de materiales.
-Funciona  nuevo item(Sin verificaciones).
-Funciona  retirarItem(verificado).
-Funciona  agregarItem(verificado).

-Modificar sin implementar.
-Salir sin implementar.

Seba328 committed:
Inventario de materiales:

-Funciona  nuevo item(verificado).
-Funciona  retirarItem(verificado).
-Funciona  agregarItem(verificado).
-Funciona  Modificar(verificado).
-Funciona  Eliminar(verificado).

-Salir sin implementar(creado Action event).

Seba328 committed:
Inventario de materiales:

-Funciona  nuevo item(verificado).
-Funciona  retirarItem(verificado).
-Funciona  agregarItem(verificado).
-Funciona  Modificar(verificado).
-Funciona  Eliminar(verificado).

-Salir sin implementar(creado Action event).

******Correccion error encapsulamiento********

mrbarrientosg commented on src/cl/loccet/model/InventarioMaterial.java:
```diff
@@ -40,6 +40,15 @@ public void retirarMaterial(Material material,int cantidad){
         material.setRetiro(cantidad);
     }
 
+    public void modificarNombre(Material material,String nombre){
```
De que me sirve guardar un ArrayList de item, si ya se que material le voy a cambiar el nombre. Lo que deberias hacer es pasar el id del material y que lo busque y reemplaze el antiguo material por el nuevo.

Podrias cambiarle el nombre a la funcion y ponerle actualizar material y pasas el material modificado

Seba328 committed:
Inventario de materiales:

-Funciona  nuevo item(verificado).
-Funciona  retirarItem(verificado).
-Funciona  agregarItem(verificado).
-Funciona  Modificar(verificado).
-Funciona  Eliminar(verificado).

-Salir sin implementar(creado Action event).

******Correccion vista table view********
******Cambio a tipo double de cantidad y retiro********
******Textfield solo admite numeros******

Seba328 committed:
Correccion de dejar campos en blancos:

-Vista retirar.
-Vista agregar.
-Vista modificar.
-Vista nuevo material.

mrbarrientosg committed:
Cambio de la vista en el fxml

mrbarrientosg committed:
Merge remote-tracking branch 'loccet/Inventario' into Inventario

# Conflicts:
#	src/cl/loccet/resource/fxml/inventariomaterial.fxml
